### ISSUE TITLE
Inconsistency in netdata.spec.in when comparing logdir permission with git-installation

### ISSUE THREAD

**Issue by 74cmonty** [netdata/netdata#4963]
##### Bug report summary
I have identified an inconsistency in [netdata.spec.in](https://github.com/netdata/netdata/blob/master/netdata.spec.in) compared to git-installation executing `kickstart.sh`.
Example:
netdata.spec.in line 214
%attr(0770,netdata,netdata) %dir %{_localstatedir}/log/%{name}

git-installation
```
# ls -l /var/log/ | grep netdata
drwxr-xr-x 2 netdata  root          4096 Dez 10 06:25 netdata
```
##### OS / Environment
any
(verified with SLES and Debian)
##### Steps To Reproduce

1. Build binary based on netdata.spec.in
2. Install this build package

##### Expected behavior
All ownership and permissions are equal comparing git-installation with build.

**Comment by cakrit** [comment]
Well, if you look at the files under the directory, you'll see that there is no inconsistency. `%attr(0770,netdata,netdata) %dir %{_localstatedir}/log/%{name}` refers to the files, not the directory. I'm not sure why root owns the directory itself, but it's not an issue, as long as the files have the correct ownership and permissions.

**Comment by 74cmonty** [comment]
When I look at the files in `/var/log/netdata` the permission is 660 (in git-installation).
In build installation the files are 644.
This means your statement is incorrect; the attribute `%attr(0770,netdata,netdata) %dir %{_localstatedir}/log/%{name}` refers to the **directory**.

**Comment by cakrit** [comment]
You're correct, let us investigate a bit further.

**Comment by cakrit** [comment]
I traced the change to the log directory ownership and permissions to #1459. The PR fixed issue #1456, related to problems with logrotate on some systems.
The relevant lines are the following: 
https://github.com/netdata/netdata/blob/master/netdata-installer.sh#L684
https://github.com/netdata/netdata/blob/master/netdata-installer.sh#L695

@paulfantom it looks like `netdata.spec.in` should be doing the same thing, i.e `%attr(0755,netdata,root)`. What do you think?
